Remove Primitive Types example

rcatalan98 opened the pull request:
Removing the file because it was in a directory that never displayed in the web page and also some parts of the example were covered by other files.

MegaRedHand reviewed (approved):
I think we should keep somewhere the part about the different ways numeric literals can be expressed in (hex, decimal, etc.).

rcatalan98 commented:
> I think we should keep somewhere the part about the different ways numeric literals can be expressed in (hex, decimal, etc.).

Good point, I just added the files to the correct folder and fixed a few issues the example had (it wasn't going to compile). We are having someone from the community doing a review/report on this project so he'll let us know if this example is useful. The more examples to evaluate the better.
